packages/prover/src/cli/cli.ts

```diff
@@ -48,6 +48,9 @@ export function getLodestarProverCli(): yargs.Argv {
     registerCommandToYargs(prover, cmd);
   }
 
+  // Register the proxy command as the default one
+  registerCommandToYargs(prover, {...proverProxyStartCommand, command: '*'});
```

Comment:
What happens if you run `lodestar-prover --help`, I would assume it no longer print out global flags and a list of available commands.

Reply:
It prints the global help screen, as before. `lodestar-prover proxy --help` prints the help for the `proxy` command as expected.